.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/node:8.11.1
+    steps:
+    - checkout
+    - run:
+        name: install packages
+        command: |
+          yarn
```

Comment:
Use postInstall hook for installing dependencies of packages

Reply:
Replaced with posthook.